### PR TITLE
[PRTL-2487] Generic Page Layout

### PR DESCRIPTION
**Pull request by yuanhengzhao**
## Ticket Number

e.g. PRTL-2487

## Environment to be used in testing

- [ ] `prod`
- [ ] `dev-oicr`
- [ ] `qa` (which version?)

## Description of Changes
Only add the MasonryLayout for ClinicalAnalysisResult usage. Can merge it into develop.

### CODE REVIEW

File: src/packages/@ncigdc/components/Layouts/MasonryLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  numPerRow: number;
+  minWidth?: number;
+  gutter?: number;
+  customizeStyle?: React.CSSProperties;
+}
+
+
+const MasonryLayout = ({
+  customizeStyle = {
+    borderStyle: 'solid', // Replace it with your own style
```
Comment (anncatton) [inline src/packages/@ncigdc/components/Layouts/MasonryLayout.tsx]: i think this comment means it's expecting a custom style prop?

File: src/packages/@ncigdc/components/Layouts/MasonryLayout.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    borderWidth: '1px',
+  },
+  elements,
+  gutter = 0.7,
+  minWidth = 354,
+  numPerRow = 3,
+}: IMasonryLayoutProps) => {
+  const width = (100 - numPerRow * gutter) / numPerRow;
```
Comment (anncatton) [inline src/packages/@ncigdc/components/Layouts/MasonryLayout.tsx]: can we name `100` so it's clear what it represents?

File: src/packages/@ncigdc/components/Layouts/MasonryLayout.tsx (hunk 1 above)
Comment (caravinci) [reply src/packages/@ncigdc/components/Layouts/MasonryLayout.tsx]: Pretty sure that's for whoever comes after (e.g. myself) to style the cards accordingly.

Comment (anncatton) [reply src/packages/@ncigdc/components/Layouts/MasonryLayout.tsx]: Lol I just totally read this prop incorrectly 🤦‍♀️Please disregard!

**Pull request merged**